packages/style-utilities/src/utilities/icons.ts

```diff
@@ -229,7 +229,7 @@ function _warnDuplicateIcon(iconName: string): void {
   if (!options.disableWarnings) {
     _missingIcons.push(iconName);
     if (_missingIconsTimer === undefined) {
-      _missingIconsTimer = setTimeout(() => {
+      _missingIconsTimer = window.setTimeout(() => {
```

Comment:
leaking global env type. if used in Node env the type will be `NodeJS.Timer` instead `number` which is not compatible. we should always use call globals with proper context (`window.`)